src/phpDocumentor/Command/Phar/UpdateCommand.php

```diff
@@ -62,63 +62,56 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $output->writeln('Looking for updates...');
 
-        $manager         = $this->createManager($output);
-        $version         = $input->getArgument('version')
-            ? $input->getArgument('version')
-            : $this->getApplication()->getVersion();
-        $allowMajor      = $input->getOption('major');
+
         $allowPreRelease = $input->getOption('pre');
 
-        if (strpos($version, 'v') === 0) {
-            // strip v tag from phar version for self-update
-            $version = str_replace('v', '', $version);
+        if (PHP_VERSION_ID < 50600) {
+            $message = 'Self updating is not available in PHP versions under 5.6.' . "\n";
+            $message .= 'The latest version can be found at ' . self::PHAR_URL;
+            $output->writeln(sprintf('<error>%s</error>', $message));
+            return 1;
+        } elseif (Application::VERSION === ('@' . 'package_version' . '@')) {
```

Comment:
Could this be written in one string '@package_version@' or is it split up for a reason?